examples/integrations/cdi/jpa/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+javax.sql.DataSource.test.dataSourceClassName=org.h2.jdbcx.JdbcDataSource
```

Comment:
Are these names required by JDBC? If not, these break the naming convention for configurable properties in Helidon (we should use lower case dash separated words, such as `javax.sql.data-source.test.data-source-class-name`.

Reply:
I am using this pattern: `<fully qualified type name>.<instance name>.<property name>`.  I do not wish to change this pattern.